test/e2e/build/build_test.go

```diff
@@ -1,4 +1,4 @@
-// +build e2e
+// +build build_e2e
```

Comment:
This disables the test, which we don't want.

Can this wait for: https://github.com/knative/serving/pull/3838?

Reply:
i'll wait for #3838